classes/Linter.js

```diff
@@ -0,0 +1,25 @@
+const { exec } = require('child_process');
+const fs = require('fs-extra');
+
+class Linter {
+  constructor() {
+    this.eslintPath = `${process.cwd()}/node_modules/.bin/eslint`;
+    this.eslintInstalled = fs.existsSync(this.eslintPath);
+
+    if (!this.eslintInstalled) {
+      throw new Error('Install eslint in the project!');
```

Comment:
should we really force usage of linter?

Reply:
It's just a constructor, so it doesn't force usage of linter. If you create an instance of this class, then eslint is required. Moreover, we use eslint so it could we useful to have such error.